garage/src/hooks/useZDK.ts

```diff
@@ -9,7 +9,7 @@ const API_ENDPOINT = "https://api.zora.co/graphql";
 const args = {
   endPoint: API_ENDPOINT,
   networks: [networkInfo],
-  // apiKey: process.env.API_KEY,
+  apiKey: import.meta.env.VITE_ZORA_API_KEY
```

Comment:
👍

Reply:
can i use this for both dev and prod (mumbai and mainnet)?

Reply:
Yeah we can for now. Zora's NFT api doesn't index mumbai but we don't need that feature right now, we will have to rethink the dev setup later if we want to load custom pilot NFTs

Reply:
cool, makes sense